guide/payments/transactions.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: guide
+title: Transaction overview
+description: An overview of Bitcoin payments, common concepts and the transaction lifecycle.
+nav_order: 1
+parent: Payments
+permalink: /guide/payments/transactions/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/overview/page-overview.svg
+---
+
+{% include picture.html
+   image = "/assets/images/guide/payments/transactions/transactions-desktop.png"
+   retina = "/assets/images/guide/payments/transactions/transactions-desktop@2x.png"
+   mobile = "/assets/images/guide/payments/transactions/transactions-mobile.png"
+   mobileRetina = "/assets/images/guide/payments/transactions/transactions-mobile@2x.png"
+   alt-text = "Process of transaction being mined, and new payments being made, mixed and sent to multiple other wallets"
+   caption = '3D payments and transaction lifecycle illustration by [Alexa Aker](https://alexaaker.com){:target="_blank" rel="nofollow"}'
+   width = 1600
+   height = 800
+   layout = "full-width"
+%}
+
+# Transactions overview
+
+This page is meant to equip you with the understanding of how bitcoin transactions get created and processed by the Bitcoin network. Bitcoin is a peer-to-peer push-payment system. This means that you can send, or *push*, bitcoin to any address at your will, at any time, without passing through a trusted third party.
+
+This is radically different from the traditional financial system, where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants, etc.). When you make a payment, it will pass through systems that might delay, control or block the payment.
+
+## Transaction lifecycle
+
+Let's lay out the entire payment process.
+
+{% include picture.html
+   image = "/assets/images/guide/payments/transactions/tx-lifecycle-simplified.svg"
+   mobile = "/assets/images/guide/payments/transactions/tx-lifecycle-simplified-mobile.svg"
+   alt-text = "Flow chart of the process of a transaction getting created and then finally confirmed"
+   width = 1350
+   height = 242
+   layout = "full-width"
+%}
+
+#### 1. Get recipient address
+The sender needs a valid address for the payment to be sent to. This can be shared by the recipient as a QR code, in plain text or as a payment link.
```

Comment:
```suggestion
The sender needs a valid address for the payment to be sent to. This can be shared by the recipient as a QR code, in plain text, or as a payment link.
```